cli/client.go

```diff
@@ -1584,9 +1624,7 @@ func outputStorageDeals(ctx context.Context, out io.Writer, full lapi.FullNode,
 
 	var deals []deal
 	for _, localDeal := range localDeals {
-		if showFailed || localDeal.State != storagemarket.StorageDealError {
```

Comment:
nit: let's remove the `showFailed` parameter from `outputStorageDeals()`

Reply:
This is done.